.github/workflows/wait_for_required_workflows.yml

```diff
@@ -78,7 +78,7 @@ jobs:
                 status: 'completed',
                 per_page: 100
               })
-              const runs = checkRuns.map(({ name, conclusion }) => ({ name, conclusion }))
+              const runs = checkRuns.map(({ id, name, conclusion }) => ({ id, name, conclusion }))
```

Comment:
I wonder why the diff shows the first id with the space after the comma, and the second id with the space before 🤷